index.html

```diff
@@ -0,0 +1,184 @@
+					<img src="./images/logo.svg" alt="logo" />
+				</figure>
+				<ul class="nav__list">
+					<li class="nav__item"><a href="#" class="nav__link">The Team</a></li>
+					<li class="nav__item"><a href="#" class="nav__link">Pricing</a></li>
+					<li class="nav__item"><a href="#" class="nav__link">Features</a></li>
+				</ul>
+			</nav>
+			<div class="header__content">
+				<div class="header__content__info">
+					<h1 class="header__heading">Beautiful Free Nova<br />template</h1>
+					<p class="header__paragraph">
+						A top notch premium quality <br />
+						template for your next web <br />
+						project.
+					</p>
+					<button class="header__btn">Download <span>free!</span></button>
+				</div>
+				<figure class="header__img">
+					<img src="./images/screen.png" alt="screen image" />
+				</figure>
+			</div>
+		</header>
+		<main>
+			<section class="features__content">
+				<div class="features__container">
+					<h2 class="features__heading">Features</h2>
+				</div>
+				<div class="features__parent">
+					<div class="features__item">
+						<div class="features__img">
+							<img src="./images/responsive.svg" alt="" />
+						</div>
+						<p class="features__paragraph">Fully responsive</p>
+						<small
+							>Looks amazing on any device:smartphone, tablet, laptop and
+							desktop.</small
+						>
+					</div>
+					<div class="features__item">
+						<div class="features__img">
+							<img src="./images/customizable.svg" alt="" />
+						</div>
+						<p class="features__paragraph">Cusomizable</p>
+						<small
+							>Change the colors, pictures or any of the sections to suit your
+							needs.</small
+						>
+					</div>
+					<div class="features__item">
+						<div class="features__img">
+							<img src="./images/design.svg" alt="" />
+						</div>
+						<p class="features__paragraph">Slick and beautiful design</p>
+						<small>trendy and fresh design, fits any website.</small>
+					</div>
+				</div>
+				<div class="features__logos">
+					<img src="./images/logos.png" alt="logos" />
+				</div>
+			</section>
+			<section class="nova__content">
+				<figure class="imac__img">
+					<img src="./images/imac.png" alt="imac computer" />
+				</figure>
+				<div class="nova__description">
+					<h3 class="nova__heading">Simple and beautiful</h3>
+					<p class="nova__paragraph">
+						Use Nova theme for your next web projects.It is completely
+						customizable so you can change any of sections to fit your needs.
+						Nova Theme is Free for any kind of use, personal and commercial.
+						Have fun and good luck!
+					</p>
+				</div>
+			</section>
+			<section class="pricing__container">
+				<div class="pricing__plan">
+					<h3 class="pricing__heading">Choose your pricing plan</h3>
+					<small class="pricing__request"
+						>Pick any of our super affordable pricing plans</small
+					>
+				</div>
+				<div class="pricing__examples">
+					<div class="pricing__basic">
```

Comment:
Każdy pakiet to osobny byt, który można odłączyć od pozostałych pakietów i będzie wiadomo czym jest - dlatego spokojnie też można użyć `article`